### PR TITLE
[exporter/loki] remove opencensus usage

### LINKED ISSUE #30170: exporter/loki

### PR DESCRIPTION
**Pull request by codeboten**
Fixes #30170

### CODE REVIEW

File: exporter/lokiexporter/exporter.go [hunk 1]
```diff
@@ -35,14 +36,23 @@ type lokiExporter struct {
...
 }
 
 func newExporter(config *Config, settings component.TelemetrySettings) *lokiExporter {
 	settings.Logger.Info("using the new Loki exporter")
 
+	count, _ := metadata.Meter(settings).Int64Counter(
+		"lokiexporter_send_failed_due_to_missing_labels",
+		metric.WithDescription("Number of log records failed to send because labels were missing"),
+		metric.WithUnit("1"),
+	)
```
Comment (bogdandrutu) [inline exporter/lokiexporter/exporter.go:50]: I see this more and more, should we actually generate an "obsrep" equivalent from metadatagen? I will give it a try maybe during the break.

File: exporter/lokiexporter/exporter.go [hunk 2]
```diff
@@ -35,14 +36,23 @@ type lokiExporter struct {
...
 	client   *http.Client
 	wg       sync.WaitGroup
+
+	lokiExporterFailedToSendLogRecordsDueToMissingLabels metric.Int64Counter
 }
 
 func newExporter(config *Config, settings component.TelemetrySettings) *lokiExporter {
 	settings.Logger.Info("using the new Loki exporter")
 
+	count, _ := metadata.Meter(settings).Int64Counter(
```
Comment (bogdandrutu) [inline exporter/lokiexporter/exporter.go:46]: I would not ignore the error if possible (a.k.a not too many changes to return error in newExporter).

**Comment by codeboten** [comment]
Pinging code owners for review: @gramidt @gouthamve @jpkrohling @mar4uk

**Comment by mar4uk** [comment]
LGTM!

File: exporter/lokiexporter/exporter.go (hunk 2 above)
Comment (codeboten) [reply exporter/lokiexporter/exporter.go:46]: added error handling

**Comment by codeboten** [comment]
@mar4uk just updated the PR to add error handling, please approve if the changes look good

**Review by mar4uk — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**